### DISCUSSION TITLE
Openlib v1.0.2

### DISCUSSION THREAD

**Discussion by dstark5** [dstark5/Openlib#27] [General]
**Changelog**

- Fixed Download Fail Due To Change In Anna's Archive Website

<hr /><em>This discussion was created from the release <a href='https://github.com/dstark5/Openlib/releases/tag/v1.0.2-beta'>Openlib v1.0.2</a>.</em>

**Comment by fallsdevil** [comment]
I liked this app a lot, it would be cool if it imported the epub files that I already have and added the dark read mode and/or plain text format to customize the size of the letters, it’s great this app! 🤩

**Reply by dstark5** [reply]
Yeah sure will add the features you mentioned in the upcoming releases 

Thank you 😊
